imports/plugins/core/graphql/server/resolvers/xforms/currency.js

```diff
@@ -0,0 +1,25 @@
+import { assoc, compose, map, toPairs } from "ramda";
+import { Reaction } from "/server/api";
+
+// add `code` and `_id` keys to each currency object
+const xformCurrencyEntry = ([k, v]) => compose(
+  assoc("code", k),
+  assoc("_id", k)
+)(v);
+
+// map over all provided currencies, provided in the format stored in our Shops collection,
+// and conver them to the format that GraphQL needs
+const xformLegacyCurrencies = compose(map(xformCurrencyEntry), toPairs);
+
+// retrive all currencies from the Shops collection and xform them
+function getXformedCurrenciesByShop() {
+  const shopCurrencies = Reaction.getShopCurrencies();
```

Comment:
I think we'll want to merge my changes that give access to Mongo outside of Meteor first, and then change this to use the collections from `context`. We need to avoid importing `Reaction`